### PR TITLE
i18n(ko-KR): update `overriding-components.md`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- add `overriding-components.md`
- I am participating in Hacktoberfest, please label it (hacktoberfest-accepted)

### CODE REVIEW

File: docs/src/content/docs/ko/guides/overriding-components.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+<h1 id="_top">{title}</h1>
+
+<style>
+  h1 {
+    font-family: 'Comic Sans';
+  }
+</style>
+```
+
+[재정의 참조](/ko/reference/overrides/#prop-types)에서 사용 가능한 모든 속성에 대해 자세히 알아보세요.
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/overriding-components.md:105]: This link should probably be updated so it matches the translated heading in #843.

```suggestion
[재정의 참조](/ko/reference/overrides/#컴포넌트-속성)에서 사용 가능한 모든 속성에 대해 자세히 알아보세요.
```

**Review by delucis — CHANGES_REQUESTED** [review]
Looks good @jsparkdev — just noticed one link we should update.

File: docs/src/content/docs/ko/guides/overriding-components.md (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/overriding-components.md]: That's a good point. thank you!

File: docs/src/content/docs/ko/guides/overriding-components.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+Starlight의 기본 UI 컴포넌트를 사용하여 나만의 컴포넌트를 구축할 수 있습니다. 즉, 사용자 정의 컴포넌트에 Starlight UI 컴포넌트를 가져와 렌더링할 수 있습니다. 이를 통해 기존 Starlight UI를 유지하면서 새로운 UI를 추가할 수 있습니다.
+
+아래 예시는 기존 `SocialLinks` 컴포넌트와 함께 이메일 링크를 렌더링하는 사용자 정의 컴포넌트를 나타냅니다.
+
+```astro {4,8}
+---
+// src/components/EmailLink.astro
+import type { Props } from '@astrojs/starlight/props';
+import Default from '@astrojs/starlight/SocialIcons.astro';
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/overriding-components.md:69]:
```suggestion
import Default from '@astrojs/starlight/components/SocialIcons.astro';
```

File: docs/src/content/docs/ko/guides/overriding-components.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+컴포넌트 재정의는 모든 페이지에 적용됩니다. 하지만, 사용자 정의 UI, Starlight의 기본 UI, 또는 완전히 다른 무언가가 나타나는 시기를 결정하기 위해 `Astro.props`의 값을 사용하여 조건부 렌더링이 가능합니다.
+
+다음 예시에서 Starlight의 [`Footer`](/ko/reference/overrides/#footer)를 재정의하는 컴포넌트는 "Starlight 🌟 를 사용하여 구축"을 홈페이지에만 표시하고, 다른 모든 페이지에는 기본 바닥글만 표시합니다.
+
+```astro
+---
+// src/components/ConditionalFooter.astro
+import type { Props } from '@astrojs/starlight/props';
+import Default from '@astrojs/starlight/Footer.astro';
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/overriding-components.md:117]:
```suggestion
import Default from '@astrojs/starlight/components/Footer.astro';
```

**Review by delucis — APPROVED** [review]
Thanks @jsparkdev — just realised we also need to add two fixes from #833. Will make those changes and get this merged 🙌

**Pull request merged**